api-reference/overview.mdx

```diff
@@ -0,0 +1,27 @@
+
+<Note>It's important to note that our API and backend SDK are intended for use exclusively in server-side applications.
+**Attempting to use them in a client-side application will result in Cross-Origin Resource Sharing (CORS) errors.**
+This restriction ensures the security and integrity of our services.</Note>
+
+
+## Authentication
+
+Authentication for the Novu API involves the use of an API Key, which is a secure credential that is tied to your Novu account. This key should be included in the header of the request in the Authorization field as a string prefixed with 'ApiKey '.
+
+For example, when using Novu in a Node.js application, the Novu package should be imported and initialized with the API key, as shown in this snippet:
+
```

Comment:
Mention the header key-value format for curl request

`Authorization: ApiKey <NOVU_API_KEY>`

Reply:
I agree with this @iampearceman